.github/workflows/release_alpha.yml

```diff
@@ -50,17 +52,17 @@ jobs:
         sudo DEBIAN_FRONTEND=noninteractive apt-get update
         sudo DEBIAN_FRONTEND=noninteractive apt-get install -yq apt-cacher-ng
 
-    - name: Get branch name
-      id: branch-name
-      uses: tj-actions/branch-names@v5
+    - name: Set major version environment variable
+      id: major-version
+      run: echo "MAJOR_VERSION=$(${{ github.event.inputs.tag }}%%.*)" >> $GITHUB_OUTPUT
```

Comment:
what's going on here?

Reply:
This line sets an environment variable named `MAJOR_VERSION` which is available in subsequent steps in the workflow. We use it to provide a build cache in the appropriate location [here](https://github.com/dashevo/dash-evo-branches/pull/33/files#diff-2dd5cabc8b0716fdda0b31b30f65114c37ffce90c8d1e0e3f1f1fbb6a0031816R65).